### PR TITLE
adds spinner

### PR DESCRIPTION
**Pull request by stoeffel**
parallel

### CODE REVIEW

**Comment by stoeffel** [comment]
going to compile coffee in parallel next

**Comment by stoeffel** [comment]
TODO

- [ ] clean up (dry) parallalCompile/compile

File: src/Builder.hs [hunk 1]
```diff
@@ -64,51 +65,131 @@ buildHelp config@Config { Config.tempDir
...
+
+symbol :: Int -> T.Text
+symbol 0 = "⣾"
+symbol 1 = "⣽"
+symbol 2 = "⣻"
+symbol 3 = "⢿"
+symbol 4 = "⡿"
+symbol 5 = "⣟"
+symbol 6 = "⣯"
+symbol _ = "⣷"
```
Comment (jwoudenberg) [inline src/Builder.hs]: 😻

**Comment by stoeffel** [comment]
going to merge this and cleanup later.

**Pull request merged**